src/pages/task/TestsTable.tsx

```diff
@@ -39,15 +40,25 @@ export const TestsTable: React.FC = () => {
       const nextQueryParams = queryString.stringify(parsed);
       replace(`${pathname}?${nextQueryParams}`);
     } else if (!validInitialQueryParams) {
+      const statuses = parsed[RequiredQueryParams.Statuses];
       setValidInitialQueryParams({
         initialCategory: parsed[RequiredQueryParams.Category],
-        initialSort: parsed[RequiredQueryParams.Sort]
+        initialSort: parsed[RequiredQueryParams.Sort],
+        initialStatuses: (Array.isArray(statuses)
+          ? statuses
+          : [statuses]
+        ).filter(v => v && v !== TestStatus.All)
```

Comment:
we discussed not needing to set initial query params a while back. instead of adding more initial query param logic, you could remove it altogether

Reply:
I made a ticket for this: https://jira.mongodb.org/browse/EVG-7611